CHANGELOG.md

```diff
@@ -4,6 +4,49 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+
+## 2.17.0 (2024-09-20)
+
+This release contains performance improvements and bug fixes since
+the 2.16.1 release. We recommend that you upgrade at the next
+available opportunity.
+
+In addition, it includes these noteworthy features:
+
+* Full PostgreSQL 17 support for all existing features
+
+
+**PostgreSQL 14 deprecation announcement**
+We will continue supporting PostgreSQL 14 until April 2025. Sooner to that time, we will announce the specific version of TimescaleDB in which PostgreSQL 14 support will not be included going forward.
+
+**Features**
+* #6882: Allow delete on the compressed chunks without decompression.
+* #7033: Use `merge` statement on continuous aggregates refresh.
+* #7126: Add functions to show the compression information.
+* #7147: Vectorize partial aggregation for `sum(int4)` with grouping on `segment by` columns.
+* #7204: Track additional extensions in telemetry.
+* #7207: Refactor the `decompress_batches_scan` functions for easier maintenance.
+* #7209: Add a function to drop the `osm` chunk.
+* #7275: Add support for the `returning` clause for `merge`.
+* #7295: Support `alter table set access method` on the hypertable.
```

Comment:
@pallavisontakke @svenklemm @mkindahl Why is this included? Should we be careful about releasing anything related to the TAM early?

For sure it should not be part of the list in the release notes even if we want to keep it around

Reply:
Agree, we can exclude this commit from the release. It is needed for the TAM eventually, but the code does not do much currently (except allowing multiple commands to `ALTER TABLE`).

Reply:
Does that mean we want to revert just this release-note while keeping the src-code commit in `main` ? As this description in-detail would get included during `2.18.0` hypercore  release.

Reply:
The commit has to be in main for a later release, but when you create the 2.17.x branch, you can remove this commit from the history. It contains a name string which might not be the final name of the table access method, so it makes sense to remove it.

Reply:
Removed the release-note for PR #7295 , as it's more a part of 2.18 story.